config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
```

Comment:
Small point, but if the user is logged in (```session[:current_user]``` is set), going to index should probably show something other than the login page

Reply:
I tried this and it seems to work:

`      def index
        if session[:current_user].nil?
            redirect_to login_path, alert: message
        else
            redirect_to controller: 'users', action: 'show', id: session[:current_user]
        end
      end`

Reply:
Yeah changed this in the user_controller file as such : 
```
def index 
        if session[:current_user]
            redirect_to controller: 'users', action: 'show', id: session[:current_user]
        end
      end
     ```
. Tested it out and it works. Thanks for catching this!